src/devices/Amg88xx/Amg88xx.cs

```diff
@@ -0,0 +1,475 @@
+        }
+
+        /// <summary>
+        /// Clears all flags in the status register
+        /// </summary>
+        public void ClearAllFlags()
+        {
+            // only the bit to be cleared is set, the other bits need to be 0
+            SetRegister(Register.SCLR, (1 << (byte)StatusClearBit.OVFCLR) | (1 << (byte)StatusClearBit.OVFTHCLR) | (1 << (byte)StatusClearBit.INTCLR));
+        }
+
+        #endregion
+
+        #region Moving average
+
+        /// <summary>
+        /// Get the state of the moving average mode
+        /// </summary>
+        /// <returns>True, if moving average mode is on</returns>
+        public bool GetMovingAverageModeState()
+        {
+            return GetBit(Register.AVE, (byte)MovingAverageModeBit.MAMOD);
+        }
+
+        /// <summary>
+        /// Sets the moving average mode state.
+        /// </summary>
+        /// <param name="state">True, to switch moving average on</param>
+        public void SetMovingAverageModeState(bool state)
+        {
+            SetBit(Register.AVE, (byte)MovingAverageModeBit.MAMOD, state);
+        }
+
+        #endregion
+
+        #region Frame Rate
+
+        /// <summary>
+        /// Get the current frame rate.
+        /// </summary>
+        /// <returns>Frame rate (either 1 or 10fps) </returns>
+        public int GetFrameRate()
+        {
+            return GetBit(Register.FPSC, (byte)FrameRateBit.FPS) ? 1 : 10;
+        }
+
+        /// <summary>
+        /// Sets the frame rate (either 1 or 10fps).
+        /// </summary>
+        /// <param name="frameRate">Frame rate</param>
+        /// <exception cref="ArgumentException">Thrown when attempting to set a frame rate other than 1 or 10</exception>
+        public void SetFrameRate(int frameRate)
+        {
+            if (frameRate != 1 && frameRate != 10)
+            {
+                throw new ArgumentException("Frame rate must either be 1 or 10.", nameof(frameRate));
+            }
+
+            SetBit(Register.FPSC, (byte)FrameRateBit.FPS, frameRate == 1);
+        }
+        #endregion
+
+        #region Operating Mode / Power Control
+
+        /// <summary>
+        /// Gets the current operating mode
+        /// </summary>
+        /// <returns>Operating mode</returns>
+        public OperatingMode GetOperatingMode()
+        {
+            return (OperatingMode)GetRegister(Register.PCLT);
+        }
+
+        /// <summary>
+        /// Sets the operating mode
+        /// </summary>
+        /// <param name="operatingMode">Operating mode</param>
+        public void SetOperatingMode(OperatingMode operatingMode)
+        {
+            SetRegister(Register.PCLT, (byte)operatingMode);
+        }
+        #endregion
+
+        #region Reset
+
+        /// <summary>
+        /// Performs an initial reset of the sensor. The flags and all configuration registers
+        /// are reset to default values.
+        /// </summary>
+        public void InitialReset()
```

Comment:
Maybe just `Rest`? would be consistent with other bindings

Reply:
I agree, we usually call it Reset or SoftwareReset

Reply:
this seems to be missed comment

Reply:
Yes, I missed it. I renamed it to "purely" ```Reset```. The datasheet calls it "initial reset", but this doesn't add anything useful for the user.